libkbfs/mdserver_remote.go

```diff
@@ -387,6 +388,20 @@ func (md *MDServerRemote) ShouldRetryOnConnect(err error) bool {
 	return !inputCanceled
 }
 
+// CheckReachability implements the MDServer interface.
+func (md *MDServerRemote) CheckReachability(ctx context.Context) {
```

Comment:
Yeah, you are right this is _only_ testing the ability to create a new connection. If the new one somehow succeeds but the old one is bad, the hope is the ping loop will detect it.